keymapper/migrations.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+# key-mapper - GUI for device specific keyboard mappings
+# Copyright (C) 2021 sezanzeb <proxima@sezanzeb.de>
+#
+# This file is part of key-mapper.
+#
+# key-mapper is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# key-mapper is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with key-mapper.  If not, see <https://www.gnu.org/licenses/>.
+
+
+"""Migration functions"""
+
+
+import os
+import json
+import copy
+
+from pathlib import Path
+
+from keymapper.logger import logger
+from keymapper.paths import get_preset_path, mkdir, CONFIG_PATH
+
+
+def all_presets():
+    p = Path(get_preset_path())
```

Comment:
using written-out variable names is better style

probably

```suggestion
    preset_path = Path(get_preset_path())
```